noxfiles/ci_nox.py

```diff
@@ -95,13 +95,19 @@ def xenon(session: nox.Session) -> None:
         "src",
         "tests",
         "scripts",
-        "--max-absolute B",
-        "--max-modules B",
-        "--max-average A",
-        "--ignore 'data, docs'",
-        "--exclude src/fides/_version.py",
+        "--max-absolute=B",
+        "--max-modules=B",
+        "--max-average=A",
+        "--ignore=data,docs",
+        "--exclude=src/fides/_version.py",
+    )
+    session.run(*command, success_codes=[0, 1])
+    session.log(
+        "Note: This command was malformed so it's been failing to report complexity issues."
+    )
+    session.log(
+        "Intentionally suppressing the error status code for now to slowly work through the issues."
     )
```

Comment:
This is the only thing we didn't discuss on Slack, let me know your thoughts on this one